data/db.js

```diff
@@ -0,0 +1,60 @@
+const knex = require('knex');
+const knexFile = require('../knexfile');
+
+const db = knex(knexFile.development);
+
+module.exports = {
+
+  getDishes: () => {
+
+    return db.select().from('dishes');
+
+  },
+
+  addDish: dish => {
+
+    return db.insert(dish).into('dishes').then(ids => { id: ids[0]});
```

Comment:
Does this (`ids => { id: ids[0]}`) actually work? Typically you need parentheses around objects that you're implicitly returning:
```javascript
ids => ({ id: ids[0]})
```
you could also shorten it down with array destructuring:
```javascript
([ id ]) => ({ id })
```